ai-platform-unified/notebooks/official/automl/tabular-model-training-and-prediction.ipynb

```diff
@@ -0,0 +1,726 @@
+{
```

Comment:
<p>Please install google-cloud-aiplatform instead of using the Github link</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/361/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Ugh sorry, that change did a runner.